app/src/main/java/cm/aptoide/pt/home/EditorialBundleViewHolder.java

```diff
@@ -74,8 +74,7 @@ private void setBundleInformation(String icon, String title, String subTitle, St
         .load(icon, backgroundImage);
     editorialTitle.setText(title);
     editorialViews.setText(String.format(itemView.getContext()
-            .getString(R.string.editorial_card_short_number_views),
-        formatNumberOfViews(numberOfViews)));
+        .getString(R.string.editorial_card_short_number_views), formatNumberOfViews(numberOfViews)));
```

Comment:
Don't know the reason of this change, I suspect it is codestyle. I went to this class, aplied the codestyle and there are changes, so it means that the codestyle is wrong here.